lib/posix/spawn/child.rb

```diff
@@ -95,6 +95,31 @@ def initialize(*args)
           @options[:pgroup] = true
         end
         @options.delete(:chdir) if @options[:chdir].nil?
+
+        @out, @err = "", ""
+
+        @stdout_stream = Proc.new do |chunk|
+          @out << chunk
+
+          true
+        end
```

Comment:
No I love it! Being able to use `method` was one of the main reasons for going back to the proc-based API ;)